daphne_worker/src/lib.rs

```diff
@@ -478,9 +487,11 @@ impl DaphneWorkerRouter {
             router
         };
 
-        let start = Date::now().as_millis();
+        // Note that we do not have a tracing span for the whole request because it typically
```

Comment:
My editor highlights "NOTE" when capitalized. 
```suggestion
        // NOTE that we do not have a tracing span for the whole request because it typically
```